### ISSUE TITLE
Show the name and size of the attached file in a non disappearing area

### ISSUE THREAD

**Issue by dominique-p** [PrivateBin/PrivateBin#783]
Request for Enhancement:
Before sending, show clearly which file is attached. 
Currently if the "Attach a file" menu is closed, one cannot see that a file is attached nor the name.
An area where the file name appears with its size would be convenient, a bit  like in Thunderbird's "Attachment" area.
It should not disappear when you type in the main Editor area. 

Regards,
Dominique

**Comment by rugk** [comment]
Good point, it would allow to double-check whether the file ios correct, etc.

**Comment by Ribas160** [comment]
To receive the attachment name and size I recommend to use the solution suggested at this comment https://github.com/PrivateBin/PrivateBin/pull/1564#discussion_r2172208261

> How about we create an attachment class with a constructor(filename, dataUrl) (we get the files as data URLs from the FileReader API when selecting and the same format is stored in the paste) and internally stores the filename, encoded and raw data, but only does the decoding if needed when calling the relevant methods, such as encoded(), raw(), size() or mimeType()? We could then pass these objects around and minimize the memory required to one or two copies.
